velox/common/encode/Base64.cpp

```diff
@@ -104,26 +117,40 @@ static_assert(
         kBase64UrlCharset,
         kBase64UrlReverseIndexTable),
     "kBase64UrlCharset has incorrect entries");
-// Similar to strchr(), but for null-terminated const strings.
-// Another difference is that we do not consider "\0" to be present in the
-// string.
-// Returns true if "str" contains the character c.
-constexpr bool constCharsetContains(
+
+/// Searches for a character within a charset up to a certain index.
```

Comment:
is it up to a certain index or starting from the index?

Reply:
updated the code